src/Okta.Sdk/Generated/GroupClient.Generated.cs

```diff
@@ -101,7 +101,7 @@ public Task<GroupRule> UpdateRuleAsync(GroupRule groupRule, string ruleId, Cance
         {
             Uri = "/api/v1/groups/rules/{ruleId}",
             Payload = groupRule,
```

Comment:
Yeah, generated files have some known whitespace issues that can't be fixed right now.